sleap/gui/app.py

```diff
@@ -812,7 +814,31 @@ def new_instance_menu_action():
 
         ### Tracks Menu ###
 
+        sessionsMenu = self.menuBar().addMenu("Sessions")
+        self.session_menu = sessionsMenu.addMenu("Set Session Instance")
+        add_menu_check_item(
+                sessionsMenu, "sync session", "Synchronize Session Settings"
+            ).setToolTip(
+                "If enabled, changes to the session settings will be synchronized across all instances."
+            )
+        add_menu_item(
+            sessionsMenu,
+            "clear session",
+            "Clear Current Session",
+            self.commands.clearCurrentSession,
+        )
+        
+        tracksMenu.addSeparator()
+
         tracksMenu = self.menuBar().addMenu("Tracks")
+        self.inst_groups_menu = sessionsMenu.addMenu("Set Instance Group")
+        add_menu_item(
+            self.inst_groups_menu,
+            "set group",
+            "Set Instance Group",
+        self.commands.setInstanceGroup 
+        )
+
```

Comment:
Let's keep the code for the Tracks Menu and Sessions Menu separate
```suggestion
        self.inst_groups_menu = sessionsMenu.addMenu("Set Instance Group")
        add_menu_item(
            self.inst_groups_menu,
            "set instance group",
            "Set Instance Group",
        self.commands.setInstanceGroup 
        )

        ### Tracks Menu ###

        tracksMenu = self.menuBar().addMenu("Tracks")
        tracksMenu.addSeparator()

```